### PR TITLE
Adding cghs.hackclub.com

### PR DESCRIPTION
**Pull request by Codaea**
# Adding `cghs.hackclub.com`

## Description

added the entry for cghs.hackclub.com

this will be the subdomain for my highschool's hack club website, where we can provide information and resources, along with links and information on how to join. 

you can check out the existing website at [cghs-hackclub.github.io](https://cghs-hackclub.github.io/). its currently in the process of being updated, and isn't in a fully functional state right now.

Thanks!

### CODE REVIEW

**Review by reesericci — CHANGES_REQUESTED** [review]
This is invalid YAML, there should only be a dash before the ttl, not before type or value

File: hackclub.com.yaml [hunk 1]
```diff
@@ -691,6 +691,10 @@ cgc:
   - ttl: 1
     type: CNAME
     value: hackclubcgc.netlify.app.
+cghs:
+  - ttl: 600
+  - type: CNAME
+  - value: cghs-hackclub.github.io
```
Comment (reesericci) [inline hackclub.com.yaml:697]:
```suggestion
    value: cghs-hackclub.github.io
```

File: hackclub.com.yaml [hunk 2]
```diff
@@ -691,6 +691,10 @@ cgc:
   - ttl: 1
     type: CNAME
     value: hackclubcgc.netlify.app.
+cghs:
+  - ttl: 600
+  - type: CNAME
```
Comment (reesericci) [inline hackclub.com.yaml:696]:
```suggestion
    type: CNAME
```

**Review by reesericci — CHANGES_REQUESTED** [review]

**Comment by Codaea** [comment]
Sorry I missed that. Thank You!

**Review by reesericci — APPROVED** [review]
LGTM

File: hackclub.com.yaml [hunk 3]
```diff
@@ -691,6 +691,10 @@ cgc:
   - ttl: 1
     type: CNAME
     value: hackclubcgc.netlify.app.
+cghs:
+  - ttl: 600
+    type: CNAME
+    value: cghs-hackclub.github.io
```
Comment (reesericci) [inline hackclub.com.yaml:697]: Missing trailing dot test says

**Review by reesericci — CHANGES_REQUESTED** [review]

File: hackclub.com.yaml (hunk 3 above)
Comment (Codaea) [reply hackclub.com.yaml:697]: gotcha

**Review by reesericci — APPROVED** [review]
LGTM

**Pull request merged**